_pages/es/roadmap.md

```diff
@@ -12,8 +12,9 @@ In this page we describe the past, current and future activities that we have on
 
 *Note that this roadmap is subject to change.*
 
-| Now (Q3 2016)  | Next (Q4 2016)   | Later (Beyond) |
+| Now (Q1 2017)  | Next (Q2 2017)   | Later (Beyond) |
 | --- | --- | --- |
-| [Dive Into Rust](/es/developer-engagement/rust-hack/) | [WebVR Camp](/es/developer-engagement/webvr-camp/) |     |
-| [Web Compatibility Sprint](/es/developer-engagement/webcompat-sprint/) | [e10s and Add-ons](/es/experiments/e10s-addons/) |     |
-| [Test Pilot Install](/es/experiments/test-pilot/) | Next-Gen Firefox Hacking with The Janitor |     |
+| [Dive Into Rust](/developer-engagement/rust-hack/) | Next-Gen Firefox Hacking with The Janitor |     |
```

Comment:
You stripped the locales prefix on the links here and all links will always lead to the English version. Is this done on purpose?